packages/api-client-core/tsconfig.cjs.json

```diff
@@ -2,7 +2,8 @@
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
     "outDir": "./dist/cjs",
-    "module": "CommonJS"
+    "module": "CommonJS",
+    "moduleResolution": "node"
```

Comment:
attw insists that module and moduleResolution match so that you aren't lying to yourself about how node will actually do things -- all these updates make that the case, which required fixing some of the busted import statements.